draft-ietf-acme-acme.md

```diff
@@ -1086,7 +1086,7 @@ Content-Type: application/problem+json
 Content-Language: en
 
 {
-  "type": "urn:ietf:params:acme:error:agreementRequired"
+  "type": "urn:ietf:params:acme:error:userActionRequired"
   "detail": "Terms of service have changed"
```

Comment:
Is there now a case beyond the terms of service changing that would require user action? Perhaps the `detail` message should be generalized slightly?